packages/build-tools/src/android/expoUpdates.ts

```diff
@@ -12,6 +12,29 @@ export enum AndroidMetadataName {
   RUNTIME_VERSION = 'expo.modules.updates.EXPO_RUNTIME_VERSION',
 }
 
+export async function androidSetRuntimeVersionNativelyAsync(
+  ctx: BuildContext<Job>,
+  runtimeVersion: string
+): Promise<void> {
+  const manifestPath = await AndroidConfig.Paths.getAndroidManifestAsync(
+    ctx.reactNativeProjectDirectory
+  );
+
+  if (!(await fs.pathExists(manifestPath))) {
+    throw new Error(`Couldn't find Android manifest at ${manifestPath}`);
+  }
+
+  const androidManifest = await AndroidConfig.Manifest.readAndroidManifestAsync(manifestPath);
+  const mainApp = AndroidConfig.Manifest.getMainApplicationOrThrow(androidManifest);
+  AndroidConfig.Manifest.addMetaDataItemToMainApplication(
+    mainApp,
+    AndroidMetadataName.RUNTIME_VERSION,
+    runtimeVersion,
+    'value'
+  );
```

Comment:
You're calling a function named `add...`. What happens if `AndroidMetadataName.RUNTIME_VERSION` already exists in the manifest?